appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 3.5.0-xprivate-{build}
+pull_requests:
+  do_not_increment_build_number: true
+skip_tags: true
+clone_depth: 1
+image: Visual Studio 2015
+init:
+- git config --global core.autocrlf true
+install:
+- git submodule update --init
+build_script:
+- ps: .\build.ps1 -Configuration Release -ReleaseLabel xprivate -BuildNumber $env:APPVEYOR_BUILD_NUMBER -SkipTests -SkipILMerge
+test_script:
+- ps: >-
+    . .\build\common.ps1
+
+    $TestErrors = @()
+
+    Invoke-BuildStep 'Running NuGet.Core tests' {
```

Comment:
Prefer custom tailored scripts rather than adding more options. It's hard to keep build.ps1 fitting all possible scenarios.